backend/build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     implementation 'org.springframework.cloud:spring-cloud-starter-loadbalancer'
 
     // Security Pins (Sept 2022)
```

Comment:
Does the comment need to be updated to April 2023

Reply:
Good question! In this case, I chose to leave it as-is since the initial `snakeyaml` pin dates back to September 2022. Removing this pin still defaults it to a vulnerable version.